types/wlr_output.c

```diff
@@ -450,6 +451,8 @@ void wlr_output_cursor_set_surface(struct wlr_output_cursor *cursor,
 }
 
 bool wlr_output_cursor_move(struct wlr_output_cursor *cursor, int x, int y) {
+	x *= cursor->output->scale;
+	y *= cursor->output->scale;
```

Comment:
Because of this, maybe we want to accept `double`s, multiply them, and cast them to `int`s right away.

Reply:
Yes, this is in my todo list